packages/core/package.json

```diff
@@ -23,12 +23,12 @@
   ],
   "scripts": {
     "clean": "rm -rf dist",
-    "build": "pnpm run clean && pnpm run build:injector && pnpm run build:injector:edge && tsup",
-    "build:injector": "tsup --config tsup.inject-standalone.config.ts",
-    "build:injector:edge": "DMNO_EDGE_COMPAT=1 tsup --config tsup.inject-standalone.config.ts",
+    "build": "pnpm run clean && pnpm run build:injector && pnpm run build:injector:edge && NODE_OPTIONS=\"--max-old-space-size=7168\" tsup",
```

Comment:
I assume this is to mitigate issues when running within the devcontainer as I have not had any issues locally.

Would it make sense to set them within the devcontainer as [`NODE_OPTIONS` env var](https://nodejs.org/api/cli.html#node_optionsoptions)?